dask_kubernetes/core.py

```diff
@@ -578,6 +578,7 @@ async def _start(self):
                     "idle_timeout": self._idle_timeout,
                     "service_wait_timeout_s": self._scheduler_service_wait_timeout,
                     "pod_template": self.scheduler_pod_template,
+                    "host": self.host,
```

Comment:
The `host` kwarg and attribute is for setting the host IP when running `deploy_mode="local"`. I think we need to add a new kwarg called something like `nodeport_host`. Perhaps we also need to update the docstring to be clear that `host` is for the local deploy mode only.